src/dragonball/src/device_manager/mod.rs

```diff
@@ -1430,7 +1430,7 @@ mod tests {
             vm.shared_info().clone(),
         );
         #[cfg(target_arch = "x86_64")]
-        let guest_addr = GuestAddress(0x200000000000);
+        let guest_addr = GuestAddress(*dbs_boot::layout::GUEST_MEM_END);
         // TODO: #7290 - https://github.com/kata-containers/kata-containers/issues/7290
         #[cfg(target_arch = "aarch64")]
         let guest_addr = GuestAddress(0xF800000000);
```

Comment:
Should we replace `0xF800000000` with `GUEST_MEM_END` for aarch64 as well?

Reply:
replaced.